src/interface/report/Results/Overview.tsx

```diff
@@ -17,20 +17,32 @@ interface Props {
 const PrototypeSwitcher = ({
   guideMode,
   setGuideMode,
+  defaultGuide,
 }: {
   guideMode: boolean;
   setGuideMode: (value: boolean) => void;
+  defaultGuide: boolean;
 }) => (
   <a style={{ justifySelf: 'end' }} href="#prototype" onClick={() => setGuideMode(!guideMode)}>
-    {guideMode ? 'Return to Normal View' : 'View Prototype'}
+    {defaultGuide
+      ? guideMode
+        ? 'View Old Version'
+        : 'Return to Guide View'
+      : guideMode
+      ? 'Return to Normal View'
+      : 'View Prototype'}
```

Comment:
nested ternaries are a little spicy but this isn't too bad.

Reply:
I hesitated before doing this I swear! But yeah I think in this particular case it's not bad